test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_inductor_training.py

```diff
@@ -0,0 +1,125 @@
+    is_pr_context,
+)
+from test.test_utils.ec2 import (
+    execute_ec2_training_performance_test,
+    ec2_performance_upload_result_to_s3_and_validate,
+    execute_ec2_habana_training_performance_test,
+)
+from src.benchmark_metrics import (
+    PYTORCH_TRAINING_GPU_SYNTHETIC_THRESHOLD,
+    PYTORCH_TRAINING_GPU_IMAGENET_THRESHOLD,
+    get_threshold_for_image,
+)
+
+PT_PERFORMANCE_TRAINING_GPU_INDUCTOR_HUGGINGFACE_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_inductor_training_performance_gpu_huggingface"
+)
+PT_PERFORMANCE_TRAINING_GPU_INDUCTOR_TIMM_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_inductor_training_performance_gpu_timm"
+)
+PT_PERFORMANCE_TRAINING_GPU_INDUCTOR_TORCHBENCH_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_inductor_training_performance_gpu_torchbench"
+)
+
+
+PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES = ("p3.2xlarge", "p4d.24xlarge", "g5.4xlarge", "g4dn.4xlarge")
+
+
+@pytest.mark.integration("inductor")
+@pytest.mark.model("huggingface")
+@pytest.mark.parametrize("ec2_instance_ami", [PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_WEST_2], indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", [PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES], indirect=True)
+def test_performance_pytorch_gpu_inductor_huggingface(pytorch_training, ec2_connection, gpu_only, py3_only):
+    execute_pytorch_gpu_py3_imagenet_ec2_training_performance_test(
+        ec2_connection, pytorch_training, PT_PERFORMANCE_TRAINING_GPU_INDUCTOR_HUGGINGFACE_CMD
+    )
+
+@pytest.mark.integration("inductor")
+@pytest.mark.model("timm")
+@pytest.mark.parametrize("ec2_instance_ami", [PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_WEST_2], indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", [PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES], indirect=True)
+def test_performance_pytorch_gpu_inductor_timm(pytorch_training, ec2_connection, gpu_only, py3_only):
+    execute_pytorch_gpu_py3_imagenet_ec2_training_performance_test(
+        ec2_connection, pytorch_training, PT_PERFORMANCE_TRAINING_GPU_INDUCTOR_TIMM_CMD
+    )
+
+@pytest.mark.integration("inductor")
+@pytest.mark.model("torchbench")
+@pytest.mark.parametrize("ec2_instance_ami", [PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_WEST_2], indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", [PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES], indirect=True)
+def test_performance_pytorch_gpu_inductor_torchbench(pytorch_training, ec2_connection, gpu_only, py3_only):
+    execute_pytorch_gpu_py3_imagenet_ec2_training_performance_test(
+        ec2_connection, pytorch_training, PT_PERFORMANCE_TRAINING_GPU_INDUCTOR_TORCHBENCH_CMD
+    )
+
+
+def execute_pytorch_gpu_py3_imagenet_ec2_training_performance_test(
+    connection, ecr_uri, test_cmd, region=DEFAULT_REGION
+):
+    _, framework_version = get_framework_and_version_from_tag(ecr_uri)
+    threshold = get_threshold_for_image(framework_version, PYTORCH_TRAINING_GPU_IMAGENET_THRESHOLD)
```

Comment:
Remove irrelevant code from this function.

Reply:
Removed.